src/coord/axisCommonTypes.ts

```diff
@@ -162,7 +163,7 @@ export interface TimeAxisBaseOption extends NumericAxisBaseOptionCommon {
     type?: 'time';
     axisLabel?: AxisLabelOption<'time'>;
 }
-interface AxisNameTextStyleOption extends TextCommonOption {
+interface AxisNameTextStyleOption extends TextCommonOption, Pick<TextStyleProps, 'overflow'| 'ellipsis'> {
```

Comment:
This should probably be `Partial<Pick<...>>` because `overflow` and `ellipsis` is optional. There should be a space before the `|`.